app/controllers/users_controller.rb

```diff
@@ -19,6 +19,10 @@ def create
 
   def show
     require_user
+    if session[:user_id]
+      @user = User.find(session[:user_id]) 
+      flash[:success] = "#{@user.name} is logged in."
+    end
```

Comment:
The conditional is just to make sure that there are no errors and no further process if there is no user.